src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.Android.cs

```diff
@@ -28,6 +28,13 @@ protected Size MeasureChildOverride(View view, Size slotSize)
 			var heightSpec = ViewHelper.SpecFromLogicalSize(slotSize.Height);
 
 			var needsForceLayout = double.IsPositiveInfinity(slotSize.Width) || double.IsPositiveInfinity(slotSize.Height);
+			if (!needsForceLayout && FeatureConfiguration.FrameworkElement.InvalidateNativeCacheOnRemeasure)
```

Comment:
Any concerns to make this the behavior without any feature flags?

Reply:
/cc @carldebilly